openbas-framework/src/main/java/io/openbas/expectation/ExpectationPropertiesConfig.java

```diff
@@ -73,8 +73,12 @@ public long getManualExpirationTime() {
   }
 
   public int getDefaultExpectationScoreValue() {
-    return ofNullable(this.defaultManualExpectationScore)
-            .orElse(DEFAULT_MANUAL_EXPECTATION_SCORE);
+    if (defaultManualExpectationScore == null ||
```

Comment:
Need to log a warning to explain that the provided value is not set because of the limitation